lib/rubocop/cop/minitest/no_test_cases.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # Checks if test class contains any test cases.
+      #
+      # @example
+      #   # bad
+      #   class FooTest < Minitest::Test
+      #     def do_something
+      #     end
+      #   end
+      #
+      #   # good
+      #   class FooTest < Minitest::Test
+      #     def test_something
+      #       assert true
+      #     end
+      #   end
+      #
+      class NoTestCases < Base
+        include MinitestExplorationHelpers
+
+        MSG = 'Test class has no test cases.'
+
+        def on_class(node)
+          return unless test_class?(node)
+
+          add_offense(node) unless test_cases(node).count.positive?
```

Comment:
Wdyt about `if test_cases(node).empty?`? 
It's interesting, that this was not already suggested by the rubocop itself.